formwidgets/blocks/assets/less/blocks.less

```
@@ -5,6 +5,68 @@
+        }
+
+        // Keep the collapse chevron's rotate-when-collapsed behaviour.
+        &.repeater-item-collapse-one {
+            transition: transform .3s, background .15s, opacity .15s;
+        }
+    }
+
+    // Rotate the collapse chevron when the item is collapsed (it now lives in
+    // the toolbar rather than the original .repeater-item-collapse wrapper).
+    .field-block-item.collapsed > .repeater-item-remove .repeater-item-collapse-one {
+        .transform(rotate(180deg));
```

Stylelint (17.12.0)

[error] Unknown word .transform (CssSyntaxError)

(CssSyntaxError)